deepchecks/utils/abstracts/confusion_matrix_abstract.py

```diff
@@ -77,3 +79,66 @@ def create_confusion_matrix_figure(confusion_matrix_data: np.ndarray, classes_na
     fig.update_yaxes(title='True value', type='category', constrain='domain', autorange='reversed')
 
     return fig
+
+
+def misclassified_samples_lower_than_condition(value: np.ndarray,
```

Comment:
see comment above - you can name this directly as add_condition_misclassified_samples_lower_than_condition